stac_geoparquet/arrow/_to_arrow.py

```diff
@@ -58,35 +43,55 @@ def parse_stac_items_to_arrow(
         downcast: if True, store bbox as float32 for memory and disk saving.
 
     Returns:
-        a pyarrow Table with the STAC-GeoParquet representation of items.
+        an iterable of pyarrow RecordBatches with the STAC-GeoParquet representation of items.
     """
+    for item_batch in batched_iter(items, chunk_size):
+        yield stac_items_to_arrow(item_batch, downcast=downcast)
```

Comment:
Don't you need to pass the `schema` along too?

Reply:
fixed in [`c935973` (#53)](https://github.com/stac-utils/stac-geoparquet/pull/53/commits/c935973db483d05815efd6f204abfbf0d7106e79)